lib/mixlib/install/backend/base.rb

```diff
@@ -27,6 +27,7 @@ class Base
         attr_reader :options
 
         SUPPORTED_WINDOWS_DESKTOP_VERSIONS = %w{7 8 8.1 10}
+        SUPPORTED_APPX_PLATFORM_VERSIONS = %w{2016}
```

Comment:
I think WIndows 8 and Windows 10 also support APPX. @mwrock Can tell us for sure.